hazelcast/src/test/java/com/hazelcast/instance/NodeThreadLeakTest.java

```diff
@@ -44,43 +44,51 @@
 /**
  * Tests that an exception in the {@link Node} and {@link NodeEngineImpl} constructor leads to properly finished services.
  */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({DefaultNodeContext.class, NodeEngineImpl.class})
-@Category(QuickTest.class)
+@RunWith(HazelcastParallelClassRunner.class)
```

Comment:
Using the default `Config config = new Config()` this starts quite a few threads, let's use serial runner here.

Reply:
Fixed: https://github.com/hazelcast/hazelcast/pull/20295/commits/0b6fda012e4580cdbef1e81ae75efc76ad3837b5